sdk/identity/azure-identity-broker/azure/identity/broker/_browser.py

```diff
@@ -32,8 +33,10 @@ class PopTokenRequestOptions(TokenRequestOptions):
 class InteractiveBrowserBrokerCredential(_InteractiveBrowserCredential):
     """Uses an authentication broker to interactively sign in a user.
 
-    Currently, only the Windows authentication broker, Web Account Manager (WAM), is supported. Users on macOS and Linux
-    will be authenticated through a browser.
+    Currently, only the following brokers are supported:
+    - Web Account Manager (WAM) on Windows
+    - Company Portal on macOS
```

Comment:
Does this mean the macOS broker can only be used by 1P customers?

Reply:
Company portal + Intune is Microsoft endpoint management solution that enables organizations to manage and secure their devices, applications, and data from a single platform.

It is not only used by Microsoft.

So I believe the answer is no. :)